infrastructure/list-api/src/main.ts

```diff
@@ -173,7 +173,7 @@ class ListAPI extends TerraformStack {
       },
       taskSize: {
         cpu: config.isDev ? 2048 : 4096,
-        memory: config.isDev ? 4096 : 8192,
+        memory: config.isDev ? 4096 : 30720,
```

Comment:
list-api also seems to suffer from a memory leak, restarting tasks about once or twice per day. Increasing the memory size should make these restarts less frequent.

![image](https://github.com/user-attachments/assets/c096c36d-14ac-454c-a8ea-c0e34ea365da)